benchmarking/manifests/jaeger-deployment.yaml

```diff
@@ -25,6 +25,7 @@ spec:
         - containerPort: 4317
```

Comment:
Please fix the commit message of `test/statistics-analysis: only pull stress-ng if not present.`
it just looks weird as it is a combination of multiple commits but without the commit message fixed.